chexus/_impl/models.py

```diff
@@ -27,13 +27,21 @@ class BucketItem(object):
             The object key of the S3 file object.
             This attribute is set with the name attribute if no key is
             provided.
+
```

Comment:
This doc can all be removed since the constructor doesn't actually accept a content_type now.